GSA/digitalgov.gov
# Topics: Merged topics #9

## Summary

Merges 30 topics.


<details>
<summary>Topics merged into new topics</summary>

| new topic            | old topic                              |
| -------------------- | ------------------------------------------------- |
| user-experience      | aoi                                               |
| software-engineering | code                                              |
| user-experience      | community-ux                                      |
| security             | cybersecurity-national-action-plan                |
| software-engineering | developer-hub                                     |
| software-engineering | devops                                            |
| search               | digitalgov-search                                 |
| user-experience      | digitalgov-user-experience-program                |
| security             | federal-risk-and-authorization-management-program |
| user-experience      | federal-user-experience-community                 |
| security             | fedramp                                           |
| usability            | first-click-testing                               |
| software-engineering | github                                            |
| software-engineering | hackathon                                         |
| security             | https                                             |
| software-engineering | ios                                               |
| software-engineering | json                                              |
| security             | national-cyber-security-awareness-month           |
| software-engineering | national-day-of-civic-hacking                     |
| software-engineering | saas                                              |
| social-media         | social-media-metrics                              |
| social-media         | socialgov                                         |
| social-media         | socialgov-community                               |
| terms-of-service     | tos                                               |
| social-media         | ugc                                               |
| social-media         | us-digital-registry                               |
| usability            | usability-case-study                              |
| usability            | usability-starter-kit                             |
| usability            | usability-testing                                 |
| user-experience      | user-experience-community-of-practice             |

</details>



> [!NOTE]
> 1 topic was removed in this PR

- [x] digitalgov-search


### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-cont-merge-topics/)



### Solution

- [x] replace old `deleted merged topic` with `new topic` in front matter
- [x] delete `deleted merged topic` folder in `content/topics`

### How To Test

1. Vist [Preview Link](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-cont-merge-topics/)
2. Review the commit messages and check that when adding that topic to the end of the preview link you get a 404 - [link to the ticket](https://trello.com/c/CVXxHb74/615-dg-topics-complete-all-merge-topics) with a spreadsheet (please note that this will start at line 566)
3. Check that the topic that is getting merged into 



### What To Test

1. Visit each old topic url and should see a 404
2. Visit replacement topic and should see old topic content displayed


---

### Dev Checklist
